source/clear-linux/tutorials/greengrass.rst

```diff
@@ -100,7 +101,7 @@ for the OpenVINO software stack:
 
    Learn more about how to :ref:`swupd-guide`.
 
-The ``computer-vision-basic`` bundle will install the OpenVINO software,
+The ``computer-vision-basic`` bundle installs the OpenVINO™ toolkit,
 along with the edge device models needed.
 
 Converting Deep Learning Models
```

Comment:
Avoid words ending in -ing 😄 . Update section heading e.g. "Convert Deep Learning Models". Also, should this be sentence case? There are several section titles using -ing in this doc, so should do a scan of the section headers checking for correct casing and to convert from using -ing where possible.